src/transformers/onnx/features.py

```diff
@@ -227,6 +229,11 @@ class FeaturesManager:
         "deit": supported_features_mapping(
             "default", "image-classification", "masked-im", onnx_config_cls="models.deit.DeiTOnnxConfig"
         ),
+        "detr": supported_features_mapping(
+            "default",
+            "object-detection",
```

Comment:
Does it make sense to include support for the segmentation head as well?

Reply:
It absolutely does, thanks for reminding me! My last commit added it. I had to set the ONNX opset at 12 to make it work.